staging/src/k8s.io/csi-translation-lib/plugins/vsphere_volume_test.go

```diff
@@ -48,6 +48,16 @@ func TestTranslatevSphereInTreeStorageClassToCSI(t *testing.T) {
 			Values: []string{"region-a"},
 		},
 	}}
+	expectedTopologySelectorTerm := v1.TopologySelectorTerm{[]v1.TopologySelectorLabelRequirement{
+		{
+			Key:    vSphereCSITopologyZoneKey,
+			Values: []string{"zone-a"},
+		},
+		{
+			Key:    vSphereCSITopologyRegionKey,
```

Comment:
Region label wasn't allowed in SC allowed topologies right?

Reply:
removing it